bin/entrypoint

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash -e
+
+# Patch /etc/passwd file with the current user info.
+# The current user's entry must be correctly defined in this file in order for
+# the `ssh` command to work within the created container.
```

Comment:
There are perfectly legit reasons to do this, but ssh inside the container isn't normally one of them (it's usually frowned upon). I'd leave that part out so as not to raise anyone else's eyebrows, and instead maybe just reference the openshift docs that explain it.

https://docs.openshift.com/container-platform/3.10/creating_images/guidelines.html#openshift-specific-guidelines